AaxDecrypter/AaxcDownloadConverter.cs

```diff
@@ -67,25 +67,40 @@ protected override bool Step2_DownloadAudiobook()
 
             aaxFile.SetDecryptionKey(downloadLicense.AudibleKey, downloadLicense.AudibleIV);
 
+            aaxFile.ConversionProgressUpdate += AaxFile_ConversionProgressUpdate;
+            //var decryptionResult = OutputFormat == OutputFormat.Mp4a ? aaxFile.ConvertToMp4a(outFile, downloadLicense.ChapterInfo) : aaxFile.ConvertToMp3(outFile);
 
-            if (File.Exists(outputFileName))
-                FileExt.SafeDelete(outputFileName);
-
-            var outputFile =  File.Open(outputFileName, FileMode.OpenOrCreate, FileAccess.ReadWrite);
+            var chapterCount = 0;
+            //aaxFile.ConvertToMultiMp4a(downloadLicense.ChapterInfo, newSplitCallback =>
+            aaxFile.ConvertToMultiMp3(downloadLicense.ChapterInfo, newSplitCallback =>
+            {
+                chapterCount++;
+                var fileName = Path.ChangeExtension(outputFileName, $"{chapterCount}.mp3");
+                if (File.Exists(fileName))
+                    FileExt.SafeDelete(fileName);
+                newSplitCallback.OutputFile = File.Open(fileName, FileMode.OpenOrCreate);
+                newSplitCallback.LameConfig = new NAudio.Lame.LameConfig
+                {
```

Comment:
When you override LameConfig, you need to specify encoding parameters. If you only want to adjust ID3 tags, don't reinitialize LameConfig. By default, MP3s are encoded to average bitrate of the input source and down sampled tomorrow.

Reply:
A couple more things on NewSplitCallback. When you call AacToMp3MultipartFilter, it initializes LameConfig with default encoding parameters and ID3 tags.  The default ID3 tags are from here:
https://github.com/Mbucari/AAXClean/blob/e7cf26757a0f311be62010d5ba2116e3579e60d6/AudioFilters/AacToMp3Filter.cs#L40

The encoding settings and tags persist between callbacks, so the minimum you need to change is the output file stream and the track number.